lib/widgets/compose_box.dart

```diff
@@ -982,6 +984,29 @@ class _FixedDestinationComposeBox extends StatefulWidget {
   State<_FixedDestinationComposeBox> createState() => _FixedDestinationComposeBoxState();
 }
 
+class _ErrorBanner extends StatelessWidget {
```

Comment:
This is separating a StatefulWidget from its corresponding State; move it above to not separate them.